main/signers/ledger/Ledger/index.ts

```diff
@@ -69,6 +64,15 @@ function getStatusForError (err: DeviceError) {
   return Status.NEEDS_RECONNECTION
 }
 
+export class DeviceError extends Error {
+  readonly statusCode
+
+  constructor (msg: string, code: number = -1) {
```

Comment:
you don't need to provide a type if you assign a default, it will be inferred by the compiler